### ISSUE TITLE
Support for v8 for `pixijs` and `@pixi/webworker` package?

### ISSUE THREAD

**Issue by gabrielecirulli** [pixijs/pixijs#9946]
### Description

There seem to be multiple distributions of pixijs:
 * `pixijs`: https://www.npmjs.com/package/pixijs (with CanvasRenderer fallback)
 * `pixi.js`: https://www.npmjs.com/package/pixi.js (WebGL only?)
 * `@pixi/webworker`: https://www.npmjs.com/package/@pixi/webworker

It's unclear to me whether v8 will be available for the Web Worker version. Will there be a beta released?

Furthermore, I'm also unsure whether the Web Worker package contains fallbacks to Canvas2D or, like `pixi.js`, only supports WebGL?

**Comment by Zyie** [comment]
Hey @gabrielecirulli 

v8 does support webworkers already. We no longer have multiple packages for pixi so `@pixi/webworker` is no longer needed

this is how it should work for v8
```ts
import { DOMAdapter, WebWorkerAdapter } from 'pixi.js';
DOMAdapter.set(WebWorkerAdapter);

... create application
```

As for canvas rendering we will be adding a brand new canvas renderer in 8.1 which should work in a webworker as well

**Comment by gabrielecirulli** [comment]
Thank you! Is the above documented somewhere?

**Comment by gabrielecirulli** [comment]
I just tested the instructions you provided using `8.0.0-rc`. Here's an excerpt of my Web Worker code. This is imported and run in my browser as a Worker instance.

```ts
import {
  Application,
  DOMAdapter,
  Graphics,
  Sprite,
  Texture,
  WebWorkerAdapter,
} from "pixi.js";

DOMAdapter.set(WebWorkerAdapter);

onmessage = async ({
  data: { serializableGameplay, width, height, resolution, canvas },
}) => {
  const app = new Application({
    width,
    height,
    resolution,
    view: canvas,
    eventMode: "none",
  });

  app.stage.addChild(buildBoardGraphic());
};

//...
```

Here's the error I'm seeing

![CleanShot 2023-12-07 at 17 37 34@2x](https://github.com/pixijs/pixijs/assets/886011/70f683a5-cf36-435a-9cdc-23da5c285adf)

**Comment by gabrielecirulli** [comment]
It looks as though something is calling `createCanvas` on BrowserAdapter during the import process.

**Comment by gabrielecirulli** [comment]
I have opened https://github.com/pixijs/pixijs/pull/9950 as an attempt to fix. I have not tested it.

**Comment by gabrielecirulli** [comment]
There are too many places where `Texture.WHITE` is used that cannot all be memoized. I am unable to think of a solution unfortunately.

**Comment by gabrielecirulli** [comment]
Might it be an idea to automatically set the right adapter as early as possible within the import process, instead of waiting on the user to do so?

**Comment by gabrielecirulli** [comment]
Hi folks, wondering if there is a known temporary workaround that I could apply to a local copy of Pixi.js in order to continue developing on v8. Otherwise, happy to wait for a fix.

**Comment by Zyie** [comment]
> Hi folks, wondering if there is a known temporary workaround that I could apply to a local copy of Pixi.js in order to continue developing on v8. Otherwise, happy to wait for a fix.

Hey, i have a PR that im working on (https://github.com/pixijs/pixijs/pull/9960), however it doesn't fully address all the issues. I'm hoping to have this finished up pretty soon though

So for right now there isn't any easy workaround

**Comment by Zyie** [comment]
hey @gabrielecirulli this should hopefully be fixed in the latest release

**Comment by gabrielecirulli** [comment]
Awesome, thank you so much! I will test it soon.
